content/venues/2022/07-27/first-thursdays-festival/readme.md

```diff
@@ -0,0 +1,10 @@
+---
+dateStart: 2022-09-01T21:00:17.165Z
```

Comment:
Date and folder names do not align. Please correct whichever is the right one.

Reply:
@Medina88 did you see this?

Reply:
corrected the folders, all should be good now.